Remove GPLv3 content so the package remains MIT licensed.

symbols/eu has been relicensed upstream.
Remove symbols/ph until that is relicensed upstream.

Upstream-Status; Backport
Signed-off-by: Ross Burton <ross.burton@intel.com>

From b3a21c1137c011d9220775fd8b2e704ff75c088c Mon Sep 17 00:00:00 2001
From: Sergey Udaltsov <sergey.udaltsov@gmail.com>
Date: Tue, 26 May 2015 20:57:46 +0100
Subject: Removed GPLv3 line

https://bugs.freedesktop.org/show_bug.cgi?id=31073

diff --git a/symbols/ph b/symbols/ph
index 6b3defa..a5fe5dc 100644
--- a/symbols/ph
+++ b/symbols/ph
@@ -2,8 +2,6 @@
 // Philippines National Keyboard Layout v1.0.0 (2010-10-23)                   //
 // http://laibcoms.com/sandbox/the-philippines-national-keyboard-layout       //
 //                                                                            //
-// License: GNU/GPLv3                                                         //
-//                                                                            //
 // Loosely based on ISO/IEC 9995-3:2009                                       //
 //	* glyphs that are not used in Philippine Languages and                //
 //		Philippine English were not included                          //
-- 
cgit v0.10.2

From 5da6d510b460cad1b31288618cc364e586576826 Mon Sep 17 00:00:00 2001
From: Sergey Udaltsov <sergey.udaltsov@gmail.com>
Date: Sat, 9 May 2015 00:35:25 +0100
Subject: fixed the license (removed GPL)

https://bugs.freedesktop.org/show_bug.cgi?id=31658

diff --git a/symbols/eu b/symbols/eu
index 0a4d31a..5b767f1 100644
--- a/symbols/eu
+++ b/symbols/eu
@@ -1,6 +1,5 @@
 // Based on <http://eurkey.steffen.bruentjen.eu/>
 // Copyright: Copyright 2012 Steffen Bruentjen (eurkey (at) steffen dot bruentjen dot de)
-// License: GPLv3 <http://www.gnu.org/licenses/gpl-3.0.html>
 // modified by Christoph Roeper <spam (at) roeper dot biz> (use firstName to mail)
 
 default  partial alphanumeric_keys modifier_keys 
-- 
cgit v0.10.2
